README.md

```diff
@@ -8,6 +8,43 @@ An internationalization library that provides locale-specific message loading, a
 
 **WARNING** This is _alpha_ software. It is not yet production ready, so you should use at your own risk.
 
+- [Usage](#usage)
+- [Features](#features)
+  - [Message Bundle Loading](#message-bundle-loading)
+  - [Determining the Current Locale](#determining-the-current-locale)
+  - [Changing the Root Locale and Observing Locale Changes](#changing-the-root-locale-and-observering-locale-changes)
+  - [Loading CLDR data](#loading-cldr-data)
+  - [ICU Message Formatting](#icu-message-formatting)
+  - [Date and number formatting](#date-and-number-formatting)
+- [How do I contribute?](#how-do-i-contribute)
+  - [Installation](#installation)
+  - [Testing](#testing)
+- [Licensing information](#licensing-information)
+
+## Usage
+
+To use `@dojo/i18n`, install the package along with its required peer dependencies:
+
+```bash
+npm install @dojo/i18n
+
+# peer dependencies
+npm install @dojo/core
+npm install @dojo/has
+npm install @dojo/shim
+```
+
+Using under TypeScript or ES6 modules, you would generally want to just import the @dojo/i18n/i18n module:
```

Comment:
Updated to `With`. We might be able to just delete that whole line though? I would think the TS example might enough for people.